APIs/openEO/Glossary.qmd

```diff
@@ -32,6 +32,7 @@ Internally it is a *process graph* with optional additional metadata.
 
 A **process graph** chains specific process calls from the set of pre-defined and user-defined processes together. A process graph itself is a (user-defined) process again. Similarly to scripts in the context of programming, process graphs organize and automate the execution of one or more processes that could alternatively be executed individually.
 
+Based on user's or creater's preference these (user-defined) process  can be shared in the [openEO algorithm plaza](https://marketplace-portal.dataspace.copernicus.eu/){target="_blank"}. You can find more information in [this page](https://documentation.dataspace.copernicus.eu/Applications/AlgorithmPlaza.html)
```

Comment:
You should also precede here with a section on parameterized user-defined processes (in pure sense: so no public or sharing aspects)

Reply:
included a sentence